src/Games/brain-calc.js

```diff
@@ -0,0 +1,44 @@
+import getRandomNumber from '../randomCount.js';
+import gamesCore from '../index.js';
+
+const descriptionOfGame = 'What is the result of the expression?';
+
+const getRandomOperation = () => {
+  const operators = ['+', '-', '*'];
+  const choiceRandomOperatorIndex = getRandomNumber(0, 2);
+  const choisedRandomOperator = operators[choiceRandomOperatorIndex];
+  return choisedRandomOperator;
+};
+
+const getResultExpression = (randomNumberOne, randomNumberTwo, randomOperator) => {
+  let resultExpression;
+  switch (randomOperator) {
+    case '+':
+      resultExpression = randomNumberOne + randomNumberTwo;
+      break;
+    case '-':
+      resultExpression = randomNumberOne - randomNumberTwo;
+      break;
+    default:
+      resultExpression = randomNumberOne * randomNumberTwo;
+      break;
+  }
+  return String(resultExpression);
```

Comment:
Is it really necessary to convert to a string?

Reply:
Yes, because answer from user - it is String. 
( if (answer === correctAnswer))
I can not use answer from user like a Number, because in first game its need be string 'yes' or 'no'